test/blackbox-tests/test-cases/all-alias/user-defined/dune

```diff
@@ -0,0 +1 @@
+(rule (with-stdout-to foo (run echo foobar)))
```

Comment:
I think that you can use `write-file` here but that should get simplified to the same.

Reply:
Actually, I'm trying to use an action that will show a trace when running `$ dune build @all --display short`. This is so that the test confirms the target was indeed built. This is a bit hacky of course, perhaps we should have a display mode that only includes the targets built.

Reply:
ah I didn't see it was `(run echo)`. that works!